frontend/ui/widget/htmlboxwidget.lua

```diff
@@ -337,11 +471,11 @@ function HtmlBoxWidget:updateHighlight()
             page:close()
         end
 
-        local text_boxes = koptinterface:getTextFromBoxes(self.page_boxes, self.hold_start_pos, self.hold_end_pos)
-        local changed = not HtmlBoxWidget:areTextBoxesEqual(self.highlight_rects, self.highlight_text, text_boxes.boxes, text_boxes.text)
+        local text, rects = HtmlBoxWidget:getSelectedText(self.page_boxes, self.hold_start_pos, self.hold_end_pos)
```

Comment:
If you are not using `self` in these utilities functions, it's best to not document/expose them as method with the `:`, and this `HtmlBoxWidget:getSelectedText()` looking non-classic :)
You can either:
- make them module local function if you guess we'll never have to use them from elsewhere
- make the class function with a `.` instead of the `:`, so you would use here `HtmlBoxWidget.getSelectedText()` (which would read more classic and we would know from reading its a class function and not a method).